microsoft-edge/toc.yml

```diff
@@ -1,327 +1,550 @@
-- name: Microsoft Edge Developer documentation #landing page
+- name: Microsoft Edge Developer documentation # landing page
   href: developer/index.yml
   items:
 # =============================================================================
     - name: Microsoft Edge DevTools
       items:
-        - name: DevTools overview
+        - name: Overview of DevTools
           href: devtools-guide-chromium/index.md
+          displayName: DevTools
+        - name: Open DevTools
+          href: devtools-guide-chromium/open/index.md
+          displayName: DevTools
         - name: What's new
           items:
             - name: Experimental features
               href: devtools-guide-chromium/experimental-features/index.md
+              displayName: Experimental features, settings
             - name: Microsoft Edge 96
               href: devtools-guide-chromium/whats-new/2021/11/devtools.md
+              displayName: Welcome tool
```

Comment:
I was not aware of the `displayName` feature. It's great!
I have a question about these What's new entries though, why use the `Welcome tool` display name for those? If I'm looking for a specific update in the TOC filter, I'm not very likely to type the word "welcome", or "tool". In fact, if I type "tool", I probably do not want those pages to appear in the list.
How about using `displayName: what's new` instead?